libraries/botbuilder-ai/src/languageTranslator.ts

```
@@ -92,7 +92,6 @@ export class LanguageTranslator implements Middleware {
                     message.text = text;
                     return Promise.resolve();
                 })
-                .catch(error => Promise.reject(error));
```

Oh yeah... Don't do that. If anything you'd want to re-throw the error.